packages/editor/src/components/widgets/table/footer/TableFooter.tsx

```diff
@@ -18,3 +18,16 @@ export const TableAddRow = (props: { colSpan: number; addRow: () => void }) => {
     </tr>
   );
 };
+
+export const TableShowMore = (props: { colSpan: number; showMore: () => void; helpertext: string }) => {
+  const readonly = useReadonly();
```

Comment:
this shouldn't be needed, because in the readonly mode you either should be able to load more rows (why should this not be allowed?) or the browser shouldn't be opeable at all...